libs/langchain-cloudflare/src/embeddings.ts

```diff
@@ -14,7 +13,7 @@ type AiTextEmbeddingsOutput = {
 
 export interface CloudflareWorkersAIEmbeddingsParams extends EmbeddingsParams {
   /** Binding */
-  binding: Fetcher;
+  binding: Ai;
```

Comment:
This will be a breaking change. Instead of this we should deprecate the `binding` input, and make it optional. With that we can add an additional (also optional) arg which is typed with the new `Ai` type.

Reply:
Hmm, good thoughts. The confusion here might come from the fact that the new AI binding is more of a binding than the old one (the old one is often referred to as a "wrapped binding" in the cf dev ecosystem), so I don't have a good name for this.

Do you have a suggestion for what this could be called? Or would it make sense to just cut a breaking change given how this is in `0.0.x` versions right now?

Reply:
Yes, let's make this a minor version.

Reply:
Makes sense to me!

Reply:
Would this also break people using this outside Cloudflare Workers?

We'll also want to update the docs but I can take care of that.

Reply:
This package is used outside of Cloudflare Workers? `Fetcher` is a purely Cloudflare Workers thing, in my experience - I had no idea this package was used externally?

Reply:
You're right, fault was mine for looking quickly.